pkg/agent/openflow/client.go

```diff
@@ -364,6 +409,9 @@ func (c *client) InstallNodeFlows(hostname string,
 		flows = append(flows, c.tunnelClassifierFlow(ipsecTunOFPort, cookie.Node))
 	}
 
+	if hardModify {
+		return c.hardModifyFlows(c.nodeFlowCache, hostname, flows)
```

Comment:
since we know NodeFlows can change, it could call `modFlows` in all cases? no need to process one particular case specifially

Reply:
Updated.